scripts/test-utils/next-env.js

```diff
@@ -1,5 +1,5 @@
 // @flow
 /* eslint-env jest */
-import serializer from 'jest-emotion'
+import serializer from 'jest-emotion/enzyme'
```

Comment:
should this include enzyme configuration? `next-env` seems like something that shouldn't